lib/services.template.ejs

```diff
@@ -31,6 +31,8 @@ var module = angular.module(<%-: moduleName | q %>, ['ngResource']);
  *
  * A $resource object for interacting with the `<%-: modelName %>` model.
  *
+ * <%-: meta['description'] %>
+ *
```

Comment:
Please make sure that multiline descriptions are correctly handled, see the description of actions (methods) for an example implementation ([link](https://github.com/strongloop/loopback-sdk-angular/blob/d170fd94e4a4fd3a36ec03e653c35ded5270a9f1/lib/services.template.ejs#L458)):

```
         * <%-: action.description | replace:/\n/g, '\n         * ' %>
```